### DISCUSSION TITLE
Custom refresh frequency at dashboard level

### DISCUSSION THREAD

**Discussion by Abhishek-kumar-ISM** [apache/superset#24486] [Ideas]
Hello! I feel that at dashboard level, there are very few options when you try `Set auto-refresh interval` option present in dashboards.

![normal](https://github.com/apache/superset/assets/120032754/457027d8-d366-41b5-8e7a-bd15372da3e0)

I think there should be some option for users to chose custom frequency based on their choice. This feature would be nice if user wants to chose frequency that is not present in options available.

This i implemented
**`https://github.com/apache/superset/pull/24449`**

It looks like as follows:
![107 99 45 216_9000_superset_dashboard_5__native_filters_key=91ksfxXf42UqpEDLp_k4NNvRNLAqehP-I5U_tTUR-qOzjqVLz4Xr-SWMRYh8cGHh (1)](https://github.com/apache/superset/assets/120032754/56e6b320-3683-4836-b56a-d2280de1e729)

![107 99 45 216_9000_superset_dashboard_5__native_filters_key=91ksfxXf42UqpEDLp_k4NNvRNLAqehP-I5U_tTUR-qOzjqVLz4Xr-SWMRYh8cGHh](https://github.com/apache/superset/assets/120032754/9f882621-f3ee-44a3-8e21-cc133214557e)

**Comment by Abhishek-kumar-ISM** [comment]
I have already implemented this feature
PR link : **https://github.com/apache/superset/pull/24449**

**Comment by lauslzrm** [comment]
Hi! @Abhishek-kumar-samsung do you know how to set a specific time for dashboard refresh (e.g., 3 AM)?

**Comment by Abhishek-kumar-ISM** [comment]
@aprendizlaura as far i know, there is no feature currently to refresh at a specific time. One can only specify refresh time interval.
